docs/src/components/landing-page/SearchQuasarForm.vue

```diff
@@ -0,0 +1,144 @@
+<template>
+  <q-form
+    v-if="showSearchInputField || isOpenByDefault"
+    autocapitalize="off"
+    autocomplete="off"
+    spellcheck="false"
+    class="search-form position-relative"
+  >
+    <transition
+      appear
+      enter-active-class="animated fadeIn"
+      leave-active-class="animated fadeOut"
+    >
+      <q-input
```

Comment:
When in input form, we need to show the same "button like" icon suggesting that pressing `/` will bring the focus to the search bar, as we have on current docs

![image](https://user-images.githubusercontent.com/10036108/151201278-0e6817e5-1911-4e10-b72f-1804a604af9d.png)

On that regards, right that feature is broken and throwns an error, you may need to adapt the listener
You can find the code in `use-search` file, it should work both when the input or the icon are shown